R/MzTabReader.R

```diff
@@ -389,6 +417,20 @@ RTUnitCorrection = function(dt)
   dt[, c(cn_rt) := lapply(.SD, function(x)  x / 60), .SDcols = cn_rt]
   #dt[, ..cn_rt]
   return(NULL)
+},
+
+rename = function(dt, namevec)
```

Comment:
--> `renameColumns()` ?
and some parameter documentation (needs to be a single string as the first line of the body .. see other R5 functions above/below)?